docker/armorblox/Dockerfile

```diff
@@ -0,0 +1,8 @@
+
+FROM demisto/python3:3.10.4.30607
+
+COPY requirements.txt .
```

Comment:
Where is this requirements.txt file?

Reply:
this requirement.txt is created by the build process using the `pipfile`. https://github.com/demisto/dockerfiles#advanced